apis/nucleus/src/hooks/useObjectSelections.js

```diff
@@ -49,6 +49,28 @@ const createObjectSelections = ({ appSelections, appModal, model }) => {
   let isActive = false;
   let hasSelected = false;
 
+  /**
+   * Event listener function on instance
+   *
+   * @method
+   * @name ObjectSelections#on
+   * @param {string} eventType event type that function needs to listen
+   * @param {Function} callback a callback function to run when event emits
+   * @example
+   * api.on('someEvent', () => {...});
+   */
+
+  /**
+   * Remove listener function on instance
+   *
+   * @method
+   * @name ObjectSelections#removeListener
+   * @param {string} eventType event type that function needs to listen
+   * @param {Function} callback a callback function to run when event emits
+   * @example
+   * api.removeListener('someEvent', () => {...});
+   */
+
```

Comment:
should it be `addListener` or `addEventListener`? 
Just asking because of we have `removeListener` without Event in it.

shouldn't both be consistent?

Reply:
Should be addListener